crates/consensus-logic/src/csm/client_transition.rs

```diff
@@ -270,12 +382,13 @@ fn handle_maturable_height(
 /// # Arguments
 ///
 /// * `state` - The client's current state, which provides the L1 view and pending checkpoints.
-/// * `checkpoints` - A slice of `BatchCheckpoint`s to be filtered.
+/// * `checkpoints` - A slice of [`BatchCheckpoint`]s to be filtered.
```

Comment:
```suggestion
/// * `checkpoints` - An array of [`BatchCheckpoint`]s to be filtered.
```

Reply:
Yeah for reading arrays are more intuitive but these are slices so keeping as it it.